src/current/v23.1/restore.md

```diff
@@ -170,10 +170,11 @@ You can exclude a table's row data from a backup using the [`exclude_data_from_b
 - All [tables](create-table.html) (which automatically includes their [indexes](indexes.html))
 - All [views](views.html)
 
-Also, a full cluster restore will:
+Also, consider that:
 
-- Restore [temporary tables](temporary-tables.html) to their original database during a full cluster restore.
-- Drop the cluster's `defaultdb` and `postgres` [pre-loaded databases](show-databases.html#preloaded-databases) before the restore begins. You can only restore `defaultdb` and `postgres` if they are present in the original [backup](take-full-and-incremental-backups.html).
+- [Temporary tables](temporary-tables.html) will be restored to their original database during a full cluster restore.
+- The restore will drop the cluster's `defaultdb` and `postgres` [pre-loaded databases](show-databases.html#preloaded-databases) before the restore begins. You can only restore `defaultdb` and `postgres` if they are present in the original [backup](take-full-and-incremental-backups.html).
+- [Changefeed jobs](change-data-capture-overview.html) will not resume automatically on the new cluster. It is necessary to manually start changefeeds after a full cluster restore.
```

Comment:
@miretskiy Should this actually be "manually create changefeeds after a full cluster restore"?

Reply:
Yes